include/api/CInferenceModelMetadata.h

```diff
@@ -49,19 +51,24 @@ class API_EXPORT CInferenceModelMetadata {
     //! Total feature importance is the mean of the magnitudes of importances for individual data points.
     void addToFeatureImportance(std::size_t i, const TVector& values);
 
+    void baseline(TVector&& baseline);
```

Comment:
This name feels too non-specific here since meta data will expand to include information distinct from feature importance.

Reply:
See [9d35bb2](https://github.com/elastic/ml-cpp/pull/1522/commits/9d35bb24e743761f526837f37661158666fccfc4).